lib/swoc/include/swoc/IPAddr.h

```diff
@@ -1388,14 +1383,12 @@ inline IPAddr::operator IP6Addr() const {
 
 inline bool
 IPAddr::operator==(self_type const &that) const {
```

Comment:
self_type has more characters than IPAddr.  Are you having a hard time staying busy in retirement?